sdk/compute/Azure.ResourceManager.Compute/tests/Mock/MockingTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.ResourceManager.Compute.Mocking;
+using Azure.ResourceManager.Compute.Models;
+using Azure.ResourceManager.Resources;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Compute.Tests.Mock
+{
+    public class MockingTests
```

Comment:
These are the mocking test cases that validate the mocking of extension methods could work as expected.